apps/backend/src/user/user.service.ts

```diff
@@ -66,4 +68,26 @@ export class UserService {
       throw new Error('Unknown username');
     });
   }
+
+  async createUsersOnStartUp(
+    userAdmin: Pick<UserDto, 'username' | 'password'>,
+    userDummyEditor: Pick<UserDto, 'username' | 'password'>
+  ) {
+    const admin = new UserEntity(userAdmin.username, hashString(userAdmin.password), Roles.ADMIN, true);
+    if (await this.isUsernameAlreadyInDatabase(admin.username)) {
+      console.log('Admin already in database.');
+    } else {
+      await this.userRepository.persistAndFlush(admin);
+    }
+    const editor = new UserEntity(userDummyEditor.username, hashString(userDummyEditor.password), Roles.EDITOR, true);
+    if (await this.isUsernameAlreadyInDatabase(editor.username)) {
+      console.log('Editor already in database.');
+    } else {
+      await this.userRepository.persistAndFlush(editor);
+    }
+  }
+
+  static async hashString(textToHash: string, rounds = 16) {
+    return await bcrypt.hash(textToHash, rounds);
```

Comment:
You should add a pinch of salt here :
https://en.wikipedia.org/wiki/Salt_(cryptography)#:~:text=In%20cryptography%2C%20a%20salt%20is,to%20safeguard%20passwords%20in%20storage.

This salt could be kept as an env variable for example :
```
PASSWORD_SALT=areYouSalty?
```

And then :
```suggestion
    return await bcrypt.hash(textToHash+process.env.PASSWORD_SALT, rounds);
```